modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -1107,16 +1136,20 @@ Type::
 |===
 | Property | Type | Description
 
+| `disableAlerts`
+| `array (string)`
+| disableAlerts is a list of alerts that should be disabled. Possible values are: `NetObservNoFlows`, which is triggered when no flows are being observed for a certain period. `NetObservLokiError`, which is triggered when flows are being dropped due to Loki errors.
```

Comment:
I wonder, if it makes sense to have separate list/section to document the alerts we currently and keep updating that page for newer alerts we may add, we can simply link it here?

Reply:
Are you saying that there's more alerts we should document besides `NetObservNoFlows` and `NetObservLokiError`? These two alerts are documenting in the [health alerts doc](https://github.com/openshift/openshift-docs/pull/56782/files#diff-47624db6ee3f3f84f0bc09f53b990e2e8c9fc0a9ffae37a39df2b28a2b455f6d) 

The simplicity of linking in product docs 🥲 is not so simple I'm afraid. I am unable to place a link here in this table cell because the structured authoring definition we have to write within. I can only place links in Assembly files after a module inclusion, so any links would be in an Additional Resources area under this API Reference table.

Reply:
no, those are the only two ATM, but was thinking if we tend more in future this "Possible values" could easily get verbose.